partner_contact_job_position/models/res_partner.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#    Copyright (c) 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+#                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+#    Copyright (c) 2015 Antiun Ingeniería S.L. (http://www.antiun.com)
+#                       Antonio Espinosa <antonioea@antiun.com>
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# © 2015 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    job_position_id = fields.Many2one(
+        "res.partner.job_position",
+        "Categorized job position",
+        oldname="job_position")
+
+
+class ResPartnerJobPosition(models.Model):
+    _name = "res.partner.job_position"
+    _order = "parent_left"
+    _parent_order = "name"
+    _parent_store = True
+    _description = "Job position"
+
+    name = fields.Char(required=True, translate=True)
+    parent_id = fields.Many2one(
```

Comment:
Well, if you use something like https://github.com/odoo/odoo/blob/af865c2e9ea1dc1890ef33add9a6e4bc657068e0/addons/hr/hr.py#L264, there's no need for the test, as this method is already tested.